src/iosched/unified.c

```diff
@@ -434,6 +477,9 @@ int unified_open(const char *path, bool open_write, struct dentry **dentry, void
 
 	ltfs_profiler_add_entry(priv->profiler, &priv->proflock, IOSCHED_REQ_ENTER(REQ_IOS_OPEN));
 	ret = ltfs_fsraw_open(path, open_write, dentry, ((struct unified_data *)iosched_handle)->vol);
+	if (!ret) {
```

Comment:
Please see my previous suggestion: at least in the I/O schedulers we're used to write if (ret == 0) to check if a function call succeeded. I think it's good idea to use the same coding pattern, as ! ret is used to check if an operation failed.

Reply:
Fixed in my branch.